apps/system/js/popup_window.js

```diff
@@ -28,6 +28,11 @@
       return;
     }
 
+    // Check if the popup is requested to be always lowered
+    if (configs.stayBackground) {
+      this.element.classList.add('alwaysLowered');
```

Comment:
we also want the `openAnimation` and `closeAnimation` to return "immediate" in this case